### ISSUE TITLE
eslint-disable-line is not working

### ISSUE THREAD

**Issue by NextSeason** [eslint/eslint#8992]
I want to set the session title of terminal by using this line in my code

`console.log( '-ne \033]0;' + settings.__module + '\007' ); // eslint-disable-line`

I got `error  Parsing error: Octal literal in strict mode` even though I added a comment `eslint-disable-line` after the line.

**Comment by not-an-aardvark** [comment]
Thanks for the report. However, this is working as intended -- from the [docs](http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments):

> Comments that disable warnings for a portion of a file tell ESLint not to report rule violations for the disabled code. ESLint still parses the entire file, however, so disabled code still needs to be syntactically valid JavaScript.

**Comment by NextSeason** [comment]
@not-an-aardvark thanks a lot, so you mean there is no way to prevent this error report unless I go to modify my code.

**Comment by not-an-aardvark** [comment]
That's correct. Your code is invalid JavaScript, so it would have caused an error when you tried to run it anyway.

**Comment by NextSeason** [comment]
@not-an-aardvark but, it is running well now, just an error report from `eslint`. If I may, can you give me any instructions about how I shall modify my code to get the same result and without the error reporting? Thanks a lot.

**Comment by not-an-aardvark** [comment]
```js
console.log( '-ne \x1b]0;' + settings.__module + '\x07' );
```

**Comment by NextSeason** [comment]
@not-an-aardvark thank you very much.
